### PR TITLE
Update Introduction to Function Graphs

### PR DESCRIPTION
**Pull request by freakin23**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [x] I have linked this PR to any issues that it closes.

### CODE REVIEW

File: content/3_Silver/Func_Graphs.mdx [hunk 1]
```diff
@@ -720,6 +720,74 @@ It's also possible to use `floyd(x)` to generate answers for all vertices in the
...
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+vector<int> res;
+vector<int> arr;
+
+void fillUp(int curr, int firstCycle) {
```
Comment (SansPapyrus683) [inline content/3_Silver/Func_Graphs.mdx:732]: 1. add comments from the java version
2. use snake_case

File: content/3_Silver/Func_Graphs.mdx [hunk 2]
```diff
@@ -720,6 +720,74 @@ It's also possible to use `floyd(x)` to generate answers for all vertices in the
...
+	arr.resize(n + 1);
+	for (int i = 1; i <= n; i++) { cin >> arr[i]; }
+
+	res.resize(n + 1, -1);
+	for (int i = 1; i <= n; i++) {
+		if (res[i] == -1) { floyd(i); }
+	}
+
+	for (int i = 1; i <= n; i++) { cout << res[i] << " \n"[i == n]; }
+	return 0;
```
Comment (SansPapyrus683) [inline content/3_Silver/Func_Graphs.mdx:786]:
```suggestion
```

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

File: content/3_Silver/Func_Graphs.mdx (hunk 1 above)
Comment (freakin23) [reply content/3_Silver/Func_Graphs.mdx]: Done.

File: content/3_Silver/Func_Graphs.mdx [hunk 3]
```diff
@@ -720,6 +720,83 @@ It's also possible to use `floyd(x)` to generate answers for all vertices in the
...
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+vector<int> res;
+vector<int> arr;
+
+/*
+ *fills up all vertices from curr to first_cycle,
+ *whose answer has already been calculated
```
Comment (SansPapyrus683) [inline content/3_Silver/Func_Graphs.mdx:734]: 💀 did you just undo my changes

Comment (freakin23) [reply content/3_Silver/Func_Graphs.mdx:734]: Sorry, I didn't saw you there. I already changed it the way you suggested.

**Review by SansPapyrus683 — APPROVED** [review]

**Comment by SansPapyrus683** [comment]
ready for merge since it's just a c++ translation of the other sols

**Pull request merged**